ansible/roles/postgres/vars/main/main.yml

```diff
@@ -0,0 +1,12 @@
+postgres_version: "15"
+postgres_daemon: "postgresql@{{ postgres_version }}-main"
+postgres_user: "postgres"
+
+postgres_users:
+  - name: pinnwand
+    password: "{{ postgres_user_passwords.pinnwand }}"
```

Comment:
Small nitpick.

We should prefix these vaulted variables with `vault_` so it's 100% clear they can't be find in any unencrypted files in the repository.

Reply:
Done

Reply:
I prefixed the `pinnwand` var name with vault ==> `vault_pinnwand`; because ansible lint complained about not having the prefix of the `postgres` role's variables being `postgres`